### ISSUE TITLE
the image of the songs interpenetrates with the title

### ISSUE THREAD

**Issue by F4bi05** [Comfy-Themes/Spicetify#122]
![Screenshot 2023-03-15 161529](https://user-images.githubusercontent.com/127969946/225355710-880c22b7-c306-45db-a046-7cefa9835ffb.png)
I don't know why but the image of the songs interpenetrates with the title, I once thought I fixed it but now I can't remember how.

**Comment by ohitstom** [comment]
that's an issue with a third party extension you are using

**Comment by F4bi05** [comment]
yes, it was a snippets that which causes the problem, weirdly it had nothing to do with the image but ok, thanks anyway.
